test/StakeManager.js

```diff
@@ -584,36 +584,21 @@ describe('StakeManager', function () {
         await assertRevert(tx5, 'staker id = 0');
       });
 
-    it('Staker should not be able to set commission if delegation not accepted', async function () {
-      const tx = stakeManager.connect(signers[1]).setCommission(5);
-      await assertRevert(tx, 'Delegetion not accpected');
+    it('Staker should not be able to accept delegation if comission is not set', async function () {
+      const tx = stakeManager.connect(signers[1]).setDelegationAcceptance('true');
+      await assertRevert(tx, 'comission not set');
     });
-
     it('Delegator should not be able to delegate if delegation not accepted', async function () {
```

Comment:
Hey 
did we miss adding this test case ?

Reply:
no all the testcases are there. but I had changed few things as per the new require condition

Reply:
cool, got it